packages/shared/src/recording/metadata/sanitizeMetadata.ts

```diff
@@ -36,9 +43,12 @@ export async function sanitizeMetadata(metadata: UnstructuredMetadata) {
           break;
         }
         default: {
-          logger.debug(
-            `Ignoring metadata key "${key}". Custom metadata blocks must be prefixed by "x-". Try "x-${key}" instead.`
-          );
+          if (opts.verbose) {
+            console.log(
```

Comment:
Just double checking that you meant to `console.log` here instead of `logger.log`

Reply:
probably not, I'll recheck this, thanks!

Reply:
This is intentional because `opts.verbose` is meant to output things into the console. Our new `logger` doesn't ever do that today

Reply:
> Our new logger doesn't ever do that today

Seems like maybe we should figure out a way to make _this_ change rather than adding one or two `console.log` statements?

Reply:
Maybe, although their purpose is somewhat different. It's one thing to log things useful *to us* and log things useful to the consumer. They won't always align. Many of our new logs are in tag/label format like:
```ts
logger.debug("LaunchBrowser:Stdout", { text });
```
From what I understand this is a format that should (eventually) be used by all of those logger calls. But with that I'm not sure what we should do when we want to both use the logger and log smth to the user. Something like this?
```ts
logger.info("SanitizeMetadata:IgnoringKey", {
  key
  verbose: verbose && `Ignoring metadata key "${key}". Custom metadata blocks must be prefixed by "x-". Try "x-${key}" instead.`
});
```

I think it's probably super bikesheddable and it could easily be put in a followup task

Reply:
Oh yeah totally. Sorry I didn't mean to suggest that was a blocker for merging this :)